views/layout.pug

```diff
@@ -45,4 +45,4 @@ body
     function fp() { fpLayer.push(arguments); }
     fp('config', 'client', 'paywall');
     fp('config', 'loaded', fpLoaded);
-  script(src="https://cdn.fpjs.io/@2/fp.js")
+  script(src="https://cdn.fingerprintjs.com/@2/fp.js")
```

Comment:
@Valve 
what domain is preferred for CDN?

Reply:
it's fpjs.io, our API and CDN is only on fpjs.io